components/omega/doc/devGuide/Eos.md

```diff
@@ -0,0 +1,25 @@
+(omega - dev -
+ eos) =
+
+    ##Equation of State
```

Comment:
This doc is inadequate. We really need to know how to use the EOS within omega with a typical use example. And we need descriptions of other public interfaces. This only describes how to create and retrieve an instance, not how to actually use it to compute spec vol

Reply:
Also, some of this does not render correctly in the preview, perhaps because of the intenting?

Reply:
@philipwjones Updated the devGuide and userGuide, please let me know if this is sufficient, or if you think more information is needed. Thanks!

Reply:
Thanks @katsmith133  The Dev guide is much better. The User guide has the necessary info but is maybe still a bit minimal and maybe jargony. Think about what you would want to see if you were a novice user of ocean models and coming to Omega for the first time - so you might want to spell out what EOS even is and then something like TEOS is recommended for real ocean simulations while linear can be used for simpler idealized test cases.

Reply:
Added these suggestions to the userGuide!